shared/libraries/opcuatms/opcuatms/src/converters/complex_number_converter.cpp

```diff
@@ -0,0 +1,77 @@
+#include "opcuatms/converters/list_conversion_utils.h"
+#include "opcuatms/converters/struct_converter.h"
+#include "opcuatms/converters/variant_converter.h"
+#include "opcuatms/core_types_utils.h"
+
+BEGIN_NAMESPACE_OPENDAQ_OPCUA_TMS
+
+// Template specializations
+
+template class StructConverter<IComplexNumber, UA_DoubleComplexNumberType>;
```

Comment:
I'd add converters for double and single precision UA complex numbers. This means defining ToDaqObject, ToVariant... for those template specifications as well.